src/index.js

```diff
@@ -12,7 +12,16 @@ import thunk from 'redux-thunk'
 
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
-const store = createStore(reducer, composeEnhancers(applyMiddleware(thunk)));
+const logger = () => {
+  return next => {
+    return action => {
+      console.log('[Middleware] Dispatching: ', action);
+      return next(action);
+    }
+  }
+}
```

Comment:
Works fine. The only thing to mention - usually all required declarations of reducers, middleware, etc. are moved to separate files/folders in order to keep this app enter point as much clean as possible.